### PR TITLE
dev-db/sqlitebrowser: add patch to fix conflict that str macro brings in

### PR DESCRIPTION
**Pull request by dpronin**
sometimes expanding str() gives an error because it conflicts with str()
 method of, for example, std::ostringstream::str()

This patch is dedicated to fixing this circumstance

### CODE REVIEW

**Comment by beatussum** [comment]
I proposed the same thing some months ago (https://bugs.gentoo.org/915385). Maybe, it would be good to add your patch to the global tree.

**Review by beatussum — APPROVED** [review]

**Comment by beatussum** [comment]
A revbump is maybe required?

**Comment by dpronin** [comment]
@beatussum what should we do to make any progress on this task?

**Comment by beatussum** [comment]
> @beatussum what should we do to make any progress on this task?

I'm not sure what to do because I'm only a proxied maintainer… Maybe, I should discuss with some Gentoo maintainer on `#gentoo` or open a new pull request under my name?

**Comment by dpronin** [comment]
[Created](https://bugs.gentoo.org/932577)

**Comment by beatussum** [comment]
> @beatussum what should we do to make any progress on this task?

I ask this to @thesamesam: I just have to allow this merge (what I do) and wait for some Gentoo developer to review it and merge it into the main tree.

**Pull request closed without merging**
